sdk/core/azure-core/inc/azure/core/internal/http/pipeline.hpp

```diff
@@ -37,8 +37,8 @@ namespace Azure { namespace Core { namespace Http { namespace _internal {
     /**
      * @brief Construct HTTP pipeline with the sequence of HTTP policies provided.
      *
-     * @param policies A sequence of #Azure::Core::Http::Policies::HttpPolicy representing a stack,
-     * first element corresponding to the top of the stack.
+     * @param policies A sequence of #Azure::Core::Http::Policies::HttpPolicy
+     * representing a stack, first element corresponding to the top of the stack.
```

Comment:
Ctrl-K-D produced that :)

Reply:
not sure. It might be a diff clang format version but still compatible with version 10.  Not sure.
If we start looking  changes to this file from one developer to another, then we need to investigate further